packages/contracts-bedrock/src/L2/SuperchainWETH.sol

```diff
@@ -42,8 +43,20 @@ contract SuperchainWETH is WETH98, IERC7802, ISemver {
     event RelayETH(address indexed from, address indexed to, uint256 amount, uint256 source);
 
     /// @notice Semantic version.
-    /// @custom:semver 1.0.0-beta.14
-    string public constant version = "1.0.0-beta.14";
+    /// @custom:semver 1.0.0-beta.13
+    string public constant version = "1.0.0-beta.13";
+
+    /// @inheritdoc WETH98
+    function deposit() public payable override {
+        if (IL1Block(Predeploys.L1_BLOCK_ATTRIBUTES).isCustomGasToken()) revert NotCustomGasToken();
```

Comment:
What is the relationship between CGT and SuperchainWETH (is it related to interoperability)?
Does it imply that an L2 with CGT enabled cannot join the Superchain ecosystem?

Reply:
I think `SuperchainWETH` has some assumptions here:
1. It is supposed to be always "wrapped eth" among the super chains.
2. The native token on L1 is supposed to be ETH.

The assumptions don't hold when CGT is enabled. I remember previously @yazhenwish said super chains can not have code change?